distribution/ddf-common/src/main/resources/etc/default.policy

```diff
@@ -0,0 +1,120 @@
+    permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read, write";
+
+    permission java.util.PropertyPermission "*", "read, write";
+};
+grant
+  codeBase "file:${ddf.home.policy}system/org/codice/-" {
+    permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read, write";
+
+    permission java.util.PropertyPermission "*", "read, write";
+};
+
+//
+// The lib directory contains root, trusted jars for DDF, OSGi, and security
+//
+grant
+  codeBase "file:${ddf.home.policy}lib/-" {
+    permission java.io.FilePermission "${ddf.home}", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}*", "read, write";
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read";
+
+    permission java.lang.RuntimePermission "createClassLoader";
+    permission java.util.PropertyPermission "*", "read, write";
+};
+grant
+  codeBase "file:${ddf.home.policy}lib/boot/-" {
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}config.properties", "write";
+};
+grant
+  codeBase "file:${ddf.home.policy}lib/boot/org.osgi.core-6.0.0.jar" {
+    // Needed to load fonts
+    permission java.io.FilePermission "<<ALL FILES>>", "read, execute";
+};
+
+//
+// Core OSGi libraries
+//
+grant
+  codeBase "file:${ddf.home.policy}system/org/eclipse/platform/org.eclipse.osgi/-" {
+    // Needed to load fonts
+    permission java.io.FilePermission "<<ALL FILES>>", "read, execute";
+
+    permission java.io.FilePermission "${ddf.home}${/}etc${/}-", "read, write";
+
+    permission java.lang.RuntimePermission "createClassLoader";
+    permission java.util.PropertyPermission "javax.net.ssl.*", "read";
+};
+
+grant {
+    permission java.util.PropertyPermission "java.*", "read";
+    permission java.util.PropertyPermission "os.name", "read";
+    permission java.util.PropertyPermission "os.version", "read";
+    permission java.util.PropertyPermission "os.arch", "read";
+    permission java.util.PropertyPermission "file.separator", "read";
+    permission java.util.PropertyPermission "path.separator", "read";
+    permission java.util.PropertyPermission "line.separator", "read";
+    permission java.util.PropertyPermission "user.*", "read";
+    permission java.util.PropertyPermission "solr.*", "read";
+
+    // User's home directory
+    permission java.io.FilePermission "${user.home}${/}-", "read, write";
+
+    // Java libraries
+    permission java.io.FilePermission "${java.home}${/}release", "read";
+    permission java.io.FilePermission "${java.home}${/}lib${/}-", "read";
+
+    // System libraries
+    permission java.io.FilePermission "/dev/urandom", "read";
+    permission java.io.FilePermission "/lib", "read";
+    permission java.io.FilePermission "/proc/self/exe", "read";
+    permission java.io.FilePermission "/usr/lib", "read";
+
+
+    // Distribution File Permissions
```

Comment:
@paouelle Wouldn't the export/import need access to the directory the exported zip was saved in?

Reply:
and a lot more. I am compiling the required changes in a separate spike.